src/LibraryViewExtensionMSWebBrowser/web/library/library.html

```diff
@@ -129,6 +162,37 @@
         //Update the view with contents the first time.
         window.external.notify("requestUpdateLibrary");
 
+        function getDPIScale() {
+            var dpi = 96.0;
+            if (window.screen.deviceXDPI != undefined) {
+                dpi = window.screen.deviceXDPI;
+            }
+            else {
+                var tmpNode = document.createElement("DIV");
+                tmpNode.style.cssText = "width:1in;height:1in;position:absolute;left:0px;top:0px;z-index:99;visibility:hidden";
+                document.body.appendChild(tmpNode);
+                dpi = parseInt(tmpNode.offsetWidth);
+                tmpNode.parentNode.removeChild(tmpNode);   
+            }
+
+            return dpi / 96.0;
+        }
+
+        function adaptDPI() {
+            document.body.style.overflowX = 'hidden';
+
+            var dpiScale = getDPIScale();
+
+            var libraryHeader = document.getElementsByClassName("LibraryHeader")[0];
+            libraryHeader.style.zoom = dpiScale;
+            
+            var widthPercentage = (100.0 / dpiScale).toString() + '%';
```

Comment:
this solution is perfectly fine to me - I think alternatively we could wrap this text in another div and scale it alone, and not scale the libraryHeader.

https://github.com/DynamoDS/librarie.js/blob/master/src/components/SearchBar.tsx#L256

But I actually like your solution more as we can isolate the changes from the librarie.js component.